ansible/roles/caso/tasks/register.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Configure cASO user
+  kolla_toolbox:
+    module_name: "kolla_keystone_user"
```

Comment:
We stopped doing it this way in Train. The module was dropped from the kolla-toolbox image. You need to use service-ks-register role now